app/history_v2_player_support.go

```diff
@@ -0,0 +1,56 @@
+package app
+
+import (
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/topfreegames/mqtt-history/logger"
+	"github.com/topfreegames/mqtt-history/mongoclient"
+
+	"github.com/labstack/echo"
+	"github.com/topfreegames/mqtt-history/models"
+)
+
+func HistoriesV2PSHandler(app *App) func(c echo.Context) error {
+	return func(c echo.Context) error {
+		c.Set("route", "HistoriesV2PlayerSupport")
+		topic := c.ParamValues()[0]
+
+		userID, playerId, _, limit, isBlocked := ParseHistoryQueryParams(c, app.Defaults.LimitOfMessages)
+		authenticated, _, err := IsAuthorized(c.StdContext(), app, userID, topic)
+		if err != nil {
+			return err
+		}
+
+		initialDateParamsFilter := c.QueryParam("initialDate")
+		finalDateParamsFilter := c.QueryParam("finalDate")
+		from := transformDate(initialDateParamsFilter)
+		to := transformDate(finalDateParamsFilter)
```

Comment:
Some things that we can do here:
1) To validate if have these parameters are passed as query strings. If not, we should send an error message, like a 400 or 422 HTTP message;
2) It could be good only to receive these dates in UTC format, to guarantee that everyone can understand the date. A good this about this is that the formatting will become easier to check. For example, instead of creating a function to parse the date you can use (like on [this example](https://go.dev/play/p/t2XhrkcYcgb)):
```go
// at the end of this file
func transformDate(dateParamsFilter string) (int64, error) {
        utcFormat := "2006-01-02" // go being go =/
	t, err := time.Parse(utcFormat, dateParamsFilter)
	if err != nil {
		return 0, err
	}
	return t.Unix()
}

//here
initialDateParamsFilter := c.QueryParam("initialDate")
from, err := transformDate(initialDateParamsFilter)
if err != nil {
	// handle err
}

finalDateParamsFilter := c.QueryParam("finalDate")
to, err := transformDate(finalDateParamsFilter)
if err != nil {
	// handle err
}
```

Reply:
thanks, Dani!